doc/user_guide/user_guide.md

```diff
@@ -1,10 +1,90 @@
 # Test DB Builder User Guide
 
-Coming soon: https://github.com/exasol/test-db-builder-java/issues/4
+Exasol's Test Database Builder for Java (TDDB) is a framework that helps you write compact and readable integration tests which require setting up a database.
 
-## Creating Users
+Instead of cluttering your test code with hundreds of lines of boilerplate code, TDDB reduces setup and population of your test database to a handful of easy to read statements. To further increase convenience TDDB supports the [fluent interface](https://en.wikipedia.org/wiki/Fluent_interface) programming style.
 
-### Creating a User who can log in
+## Preparations
+
+The first step of any integration test involving a database is installing that database. In case of Exasol, we recommend you simplify your life by letting [Exasol Testcontainers](https://github.com/exasol/exasol-testcontainers) do that for you. This will give you a disposable Exasol installation running inside a [Docker](https://www.docker.com/) container. Just the thing you need for integration testing.
```

Comment:
```suggestion
The first step of any integration test involving a database is installing that database. In the case of Exasol, we recommend you simplify your life by letting [Exasol Testcontainers](https://github.com/exasol/exasol-testcontainers) do that for you. This will give you a disposable Exasol installation running inside a [Docker](https://www.docker.com/) container. Just the thing you need for integration testing.
```